setup.py

```diff
@@ -31,18 +31,21 @@
     ],
     language="c++",
     extra_compile_args=extra_compile_args,
-    include_dirs=[numpy_include_dir, "include"]
+    include_dirs=[numpy_include_dir, os.path.join('morphsnakes', 'include')]
 )
 
 setup(name="MorphSnakes",
-    version="1.0.0",
+    version="1.0.1",
     description="Morphological Snakes",
     author="Pablo Márquez Neila",
     author_email="pablo.marquezneila@epfl.ch",
-    url="https://github.com/pmneila/morphsnakes",
+    url="https://github.com/borda/morphsnakes",
```

Comment:
Please, don't do this, it's not nice. Keep the URL of the project to the original one.

Reply:
ok, I will change it back, this was just because you did not have the option of multiple snakes in parallel and I was not able to compile your original C version